concepts/security/README.md

```diff
@@ -33,28 +33,45 @@ Dapr also supports strong identities when deployed on Kubernetes, relying on a p
 By default, a workload cert is valid for 24 hours and the clock skew is set to 15 minutes.
 
 Mutual TLS can be turned off/on by editing the default configuration that is deployed with Dapr via the `spec.mtls.enabled` field.
-This can be done for both Kubernetes and self hosted modes.
-Specific details for how to do that can be found [here](../../howto/configure-mtls).
+This can be done for both Kubernetes and self hosted modes. Details for how to do this can be found [here](../../howto/configure-mtls).
+
+## Sidecar to Control Plane communication
+
+In addition to automatic mTLS between Dapr sidecars, Dapr offers mandatory mTLS between the Dapr sidecar and the Dapr system services (Control Plane), namely Sentry service (Certificate Authority), Placement service (actor placement) and the Kubernetes Operator.
+
+When mTLS is enabled, Sentry writes the root and issuer certificates to a Kubernetes secret that is scoped to the namespace where the control plane is installed in. In self hosted mode, Sentry writes the certificates to a configurable filesystem path.
+
+In Kubernetes, when the Dapr system services start, they automatically mount the secret containing the root and issuer certs and use those to secure the gRPC server that is used by the Dapr sidecar.  
+
+In self hosted mode, each system service can be configured with the filesystem path to mount the credentials from.
+
+When the Dapr sidecars initialize, they authenticate with the system services using the workload cert that was issued to them by Sentry, the Certificate Authority.
 
 ### mTLS in Kubernetes
 
-<a href="https://ibb.co/4VXWJ3d"><img src="https://i.ibb.co/rwzkvNs/Screen-Shot-2020-02-10-at-8-35-59-PM.png" alt="Screen-Shot-2020-02-10-at-8-35-59-PM" border="0"></a>
+<a href="https://ibb.co/NN1Ycsj"><img src="https://i.ibb.co/s15382w/Screen-Shot-2020-03-30-at-6-22-04-PM.png" alt="Screen-Shot-2020-03-30-at-6-22-04-PM" border="0"></a>
```

Comment:
This Kubernetes file image is wrong. It is not the same as slide 13 in the diagrams deck. Text is missing